pkg/node/volumemgr.go

```diff
@@ -699,7 +706,7 @@ func (m *VolumeManager) discoverVolumeCRs() error {
 		"method": "discoverVolumeCRs",
 	})
 
-	driveCRs, err := m.crHelper.GetDriveCRs(m.nodeID)
+	driveCRs, err := m.cachedCrHelper.GetDriveCRs(m.nodeID)
```

Comment:
Disk can be marked as OFFLINE in controller **statemonitor.go**. Is this really safe to read from cache here?

Reply:
Cache is autoupdated, so we will receive this update.